### ISSUE TITLE
Link external in Nav

### ISSUE THREAD

**Issue by over12** [retypeapp/retype#236]
Hi, I saw that you have ways to add external links in the Top and Footer of the site, but I would like to know if it is possible to add external links in the main navigation menu?

I'm organizing my whitepaper and it would look much more organized the way I'm doing it (print attached).

![image](https://user-images.githubusercontent.com/12565400/145366734-7be7b7d6-b43b-499f-b0ba-91b2f2fdd9e3.png)

**Comment by fabriciomurta** [comment]
This is not the first time one asked about this, but I couldn't find the answer we posted in the other thread in a quick search, so here you go:

- No. Retype does not "support" linking from the left nav bar.
- Retype supports though raw HTML enabling you to do this:

**facebook.md**
```markdown
# Facebook

<meta http-equiv="refresh" content="0;URL='https://www.facebook.com/'" />
```

It causes a bit of a "blink" as the page needs to be loaded before the meta kicks in, but it should work in most browsers.

**Comment by fabriciomurta** [comment]
Oops, we have implemented this since version 1.8 and forgot to place a follow-up here!

We have, back then, implemented [the `redirect` page config](https://retype.com/configuration/page/#redirect) which does exactly what you wanted, inspired on your very request! Sorry for the long time, and let us know if that's not really what you wanted.
